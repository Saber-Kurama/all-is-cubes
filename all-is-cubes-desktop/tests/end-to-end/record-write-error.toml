# Tests what happens if the output file cannot be written
# (in this case because it's an existing directory).
# This isn't strongly an "end-to-end" test; we're just taking advantage
# of trycmd's ability to set up input and output files.
bin.name = "all-is-cubes"
args = "--no-config-files --simplify-log-format --template lighting-bench --seed 0 --graphics record -o obstacle.png --display-size 120x90"
status.code = 1
