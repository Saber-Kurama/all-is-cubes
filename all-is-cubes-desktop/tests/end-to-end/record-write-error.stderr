[..] Initialized session ([..] s)
[..] Initialized game state with Template(LightingBench, TemplateParameters { seed: Some(0), size: None }) ([..] s)
Error: failed to configure session for recording

Caused by:
    [..]
